### PR TITLE
Update guided pages with NWB GUIDE content

### LINKED ISSUE #61: Update text and icons on main page and guided mode
Since I've assigned #60 and #59 to Ryan, I'll assign the current in-development pages to @garrettmflynn 

Can you update the text and assets for the main page and first couple pages for guided mode to be more NWB-like and less SODA?

I think you had previously done this at some point anyway but maybe it's still split off?

### PR DESCRIPTION
**Pull request by garrettmflynn**
fix #61 

This PR updates Guided Mode pages with GUIDE-related content. Review [Build 39](https://www.chromatic.com/build?appId=640a4d93e34604f275368983&number=39) for changes.

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
Requested style changes

image

File: src/stories/pages/guided-mode/GuidedHome.js [hunk 1]
```diff
@@ -219,18 +219,15 @@ export class GuidedHomePage extends Page {
...
-              standards is required. Contrary to the Free Form Mode, the Guided Mode interfaces are
-              interconnected and form a single workflow such that the curation process is streamlined
-              further.
+          <h1 class="guided--text-sub-step">Guided Mode</h1>
+          <p class="guided--help-text" style="margin-bottom: 2rem">
+              NWB GUIDE is intended to guide users step-by-step through all the requirements for
+              converting and uploading datasets to the NWB format for the DANDI Archive. The user interfaces
+              of the Guided Mode are designed to logically guide users through the conversion process and
+              include all necessary information such that no prior knowledge of the NWB data
+              standard is required.
```
Comment (CodyCBakerPhD) [inline src/stories/pages/guided-mode/GuidedHome.js:228]: Quick side question: in JS, is it common to break paragraphs over multiple lines of code, or have one long line + and IDE that handles soft wrapping?

Comment (CodyCBakerPhD) [inline src/stories/pages/guided-mode/GuidedHome.js:228]:
```suggestion
              The NWB GUIDE will walk users step-by-step through all the requirements for
              converting and uploading datasets to the NWB format for the DANDI Archive. The user interfaces
              of the Guided Mode are designed to logically guide users through the conversion process and
              include all necessary information such that no prior knowledge of the NWB data
              standard is required.
```

when reading out loud, reduces the number of times 'guide' is repeated

Comment (CodyCBakerPhD) [inline src/stories/pages/guided-mode/GuidedHome.js:228]:
```suggestion
              NWB GUIDE is intended to guide users step-by-step through all the requirements for
              converting their data to the NWB format and uploading datasets to the DANDI Archive. The user interfaces
              of the Guided Mode are designed to logically guide users through the conversion process and
              include all necessary information such that no prior knowledge of the NWB data
              standard is required.
```

Reading was slightly awkward due to the phrase 'uploading datasets to the NWB format' - decoupled each step to more accurately reflect the workflow

Comment (CodyCBakerPhD) [inline src/stories/pages/guided-mode/GuidedHome.js:228]:
```suggestion
              NWB GUIDE is intended to guide users step-by-step through all the requirements for
              converting and uploading datasets to the NWB format for the DANDI Archive. Each stage
              of the Guided Mode is designed to conveniently direct users through the conversion process and
              include all necessary information such that no prior knowledge of the NWB data
              standard is required.
```

What do you think of this rephrasing? Trying to reduce repetition of 'user' and 'guide'

File: src/stories/pages/guided-mode/GuidedStart.js [hunk 2]
```diff
@@ -66,31 +66,29 @@ export class GuidedStartPage extends Page {
             </div>
             <div class="guided--section">
               <p class="guided--help-text">
-                Guided Mode is designed to be the simplest and most efficient method of curating and
-                sharing a SPARC dataset if you are starting from scratch. Guided Mode guides you
-                through the entire SPARC curation and sharing process via intuitive user interfaces
+                Guided Mode is designed to be the simplest and most efficient method of converting to the NWB format and uploading to the DANDI Archive if you are starting from scratch. Guided Mode guides you
+                through the entire conversion process via intuitive user interfaces
```
Comment (CodyCBakerPhD) [inline src/stories/pages/guided-mode/GuidedStart.js:70]: Hmm.. Given how good the summary of Guided Mode is on the previous page, do you think we need this paragraph here?

File: src/stories/pages/guided-mode/GuidedStart.js [hunk 3]
```diff
@@ -66,31 +66,29 @@ export class GuidedStartPage extends Page {
...
               </div>
               <p class="guided--help-text" style="margin-top: 1em">
-                Guided Moded is divived in four high-level sections. During the first three sections,
-                you will be directed to specify the files to include in your dataset and provide
-                information about your dataset. On the final section, SODA will automatically generate
-                your dataset on Pennsieve with the proper folder structure and metadata files. Note
+                Guided Mode is divided in four high-level sections. During the first three sections,
+                you will be directed to specify the data formats and files to include in your dataset and provide
+                information about your dataset. On the final section, NWB GUIDE will automatically generate
+                a valid NWB file and ask for your review before uploading to DANDI. Note
```
Comment (CodyCBakerPhD) [inline src/stories/pages/guided-mode/GuidedStart.js:91]:
```suggestion
                Guided Mode is divided into four high-level sections. During the first three sections,
                you will be directed to specify the data formats and files to include in your dataset and provide
                information about your dataset. In the final section, NWB GUIDE will automatically generate
                a valid NWB file and ask for your review before uploading to DANDI. Note
```

**Comment by CodyCBakerPhD** [comment]
Huh, what's really interesting is that those animations on the main page get flagged as 'changes' by Chromatic whenever it takes screenshots

Is there maybe a way to set those to be static only for Chromatic upload?

File: src/stories/pages/guided-mode/GuidedHome.js (hunk 1 above)
Comment (garrettmflynn) [reply src/stories/pages/guided-mode/GuidedHome.js:228]: Generally, I'd say a typical workflow for writing HTML is to keep one long line or, depending on the programmer, break your paragraph into separate lines for immediate readability—then run a code formatter (e.g. prettier) to handle the breaks in a standard fashion, which I think happened here.

In HTML, it doesn't really matter how line breaks are formatted inside the p tag, as they'll be rendered without them anyways.

File: src/stories/pages/guided-mode/GuidedStart.js (hunk 2 above)
Comment (garrettmflynn) [reply src/stories/pages/guided-mode/GuidedStart.js:70]: Yeah, I'd agree we probably don't need this. 

Honestly, we probably don't need this page at all—though the table of contents-like paragraph after the dropdown information is nice to have.

What do you think? Should we keep this page? And if we do, what about that dropdown with more info about neuroconv?

**Comment by CodyCBakerPhD** [comment]
Many merge conflicts resulting from SODA purge - at least that shouldn't be a problem moving forward 😕

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**